frontend/packages/topology/src/filters/FilterDropdown.tsx

```diff
@@ -44,6 +49,9 @@ const FilterDropdown: React.FC<FilterDropdownProps> = ({
       value,
     };
     onChange([...filters.slice(0, index), filter, ...filters.slice(index + 1)]);
+    fireTelemetryEvent('Topology Display Option Changed', {
+      [EXPAND_GROUPS_FILTER_ID]: value,
+    });
```

Comment:
So in the excel sheet it says to return the property name and its value separately. Though this also looks fine, it will be better to confirm.

Reply:
i missed it , will have it same as in sheet